cmd/lotus-miner/info.go

```diff
@@ -126,7 +126,7 @@ func infoCmdAct(cctx *cli.Context) error {
 
 	alerts, err := minerApi.LogAlerts(ctx)
 	if err != nil {
-		return xerrors.Errorf("getting alerts: %w", err)
+		fmt.Printf("ERROR: getting alerts: %w\n", err)
```

Comment:
Should be
```suggestion
		fmt.Printf("ERROR: getting alerts: %s\n", err)
```
(`%w` is only valid in `Errorf`)

Reply:
Yes, that's far better.  Thank you!

```
$ lotus-miner info | head -n 10
Enabled subsystems (from miner API): [Mining Sealing SectorStorage Markets]
Enabled subsystems (from markets API): [Mining Sealing SectorStorage Markets]
Chain: [sync ok] [basefee 100 aFIL]
ERROR: getting alerts: missing permission to invoke 'LogAlerts' (need 'admin')
Miner: t01105 (32 GiB sectors)
Power: 1.19 Ti / 2.59 Pi (0.0448%)
        Raw: 1.188 TiB / 1.956 PiB (0.0593%)
        Committed: 1.188 TiB
        Proving: 1.188 TiB
Projected average block win rate: 45.10/week (every 3h43m29s)
$
```